tests/test_object_detection.py

```diff
@@ -76,8 +91,9 @@ def test_inference_autocast(self):
             target_sizes = torch.Tensor([image.size[::-1]])
             results = processor.post_process_object_detection(outputs, target_sizes=target_sizes, threshold=0.9)[0]
             boxes = results["boxes"]
-            self.assertEqual(len(boxes), 5)
-            expected_location = np.array([342, 25.25, 636, 376])
+            self.assertEqual(len(boxes), self.get_expected_num_boxes())
+            print(f"boxes = {boxes}")
```

Comment:
please remove debug print, it might be more meaningful if added as part of the assertLess(first, second, msg="<your message on boxes value>")